packages/backend/src/open_payments/quote/service.ts

```diff
@@ -347,13 +357,36 @@ function calculateFixedSendQuoteAmounts(
   }
 }
 
+function calculateExpiry(
+  deps: ServiceDependencies,
+  quote: UnfinalizedQuote,
+  receiver: Receiver
+): Date {
+  const quoteExpiry = new Date(Date.now() + deps.config.quoteLifespan)
```

Comment:
creates an expiry of now, instead of `createdAt` (since we are setting ahead of creation). This was to allow us to simply create the quote and not create then update it.